scripts/travis-setup.sh

```diff
@@ -1,28 +1,20 @@
 #!/bin/bash
-set -e
-
-SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+set -euo pipefail
 
 if [[ $TRAVIS_OS_NAME = "linux" ]]
 then
-    docker pull bioconda/bioconda-builder
-else
-
-    # install conda
-    curl -O https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
-    sudo bash Miniconda3-latest-MacOSX-x86_64.sh -b -p /anaconda
-    sudo chown -R $USER /anaconda
-    mkdir -p /anaconda/conda-bld/osx-64 # workaround for bug in current conda
-    mkdir -p /anaconda/conda-bld/linux-64 # workaround for bug in current conda
-    export PATH=/anaconda/bin:$PATH
-    conda install -y --file $SCRIPT_DIR/requirements.txt
-    conda index /anaconda/conda-bld/linux-64 /anaconda/conda-bld/osx-64
+    sudo apt-get update -qq
+    sudo apt-get install docker-engine --no-install-recommends -y -o Dpkg::Options::="--force-confmiss" -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confnew"
```

Comment:
Yes it does, this was part of my debugging. This will install the latest docker version on Linux. If you think we should stick with an older version, I can remove it.